ui/design-system-core/src/components/Icon/index.tsx

```diff
@@ -1,16 +1,27 @@
 import React from 'react';
 import * as HeroIcons from '@heroicons/react/24/outline';
+import * as SolidHeroIcons from '@heroicons/react/24/solid';
 
+export type icontype = 'solid' | 'outline';
 export type IconName = keyof typeof HeroIcons;
-interface IconProps {
+export interface IconProps {
   icon: IconName;
   styling?: string;
+  strokeColor?: string;
+  fillColor?: string;
+  iconType?: icontype;
 }
 
-const Icon: React.FC<IconProps> = props => {
-  const PassedIcon = HeroIcons[props.icon];
+const Icon: React.FC<IconProps> = ({
+  icon,
+  styling,
+  strokeColor = 'currentColor',
+  fillColor = 'currentColor',
+  iconType = 'outline',
+}) => {
+  const PassedIcon = iconType === 'outline' ? HeroIcons[icon] : SolidHeroIcons[icon];
```

Comment:
This too, also consider the custom Icons when resolving from next 👍

Reply:
noted